src/Type.zig

```diff
@@ -1020,7 +1020,7 @@ pub fn abiAlignmentInner(
             },
 
             // represents machine code; not a pointer
-            .func_type => return .{ .scalar = target_util.defaultFunctionAlignment(target) },
+            .func_type => return .{ .scalar = target_util.minFunctionAlignment(target) },
```

Comment:
It didn't seem to me like this value had any practical significance, so simply using the ABI minimum should be fine?

Reply:
This is `@alignOf(fn ...)`, so is relevant to things like function pointer coercions to aligned pointers. I think it makes sense to use the ABI minimum, since that's all we can guarantee if it's up to the optimizer how functions are aligned in practice.